client/dist/style.css

```diff
@@ -2,6 +2,15 @@ body{
   background-color: #041c2c;
 }
 
+.main_title{
```

Comment:
You should use CSS Modules or StyledComponents asap.

Reply:
@therobinkim Converting to StyledComponents now!